app/src/androidTest/java/org/mozilla/tv/firefox/helpers/SessionLoadedIdlingResource.kt

```diff
@@ -25,9 +26,10 @@ class SessionLoadedIdlingResource : IdlingResource {
 
         val session = sessionManager.selectedSession
 
-        return if (session?.loading == true) {
+        return if (session?.loading == true && !ignoreLoading) {
             false
         } else {
+            ignoreLoading = false
             invokeCallback()
             true
```

Comment:
Does this work consistently?  It seems like this is going to cause a race condition.  If `isIdleNow` is checked twice before a command is executed, it will block on `session.loading`.